sujeet/question2.txt

```diff
@@ -0,0 +1,6 @@
+Q2. How to set multiple remote repositories for the same project? Explain the use case. you should have a working demo of it.
+
+Ans: git remote add origin (url);
+     git add upstream (url)
```

Comment:
Please add stepwise commands for how to create multiple repos on different platforms